src/extensions/extension-loader/extension-loader.ts

```diff
@@ -339,9 +338,16 @@ export class ExtensionLoader {
       // Remove null values
       .filter(extension => Boolean(extension));
 
-    // We first need to wait until each extension's `onActivate` is resolved,
+    // We first need to wait until each extension's `onActivate` is resolved or rejected,
     // as this might register new catalog categories. Afterwards we can safely .enable the extension.
-    await Promise.all(extensions.map(extension => extension.activated));
+    await Promise.all(
+      extensions.map(extension =>
+        // If extension activation fails, log error
+        extension.activated.catch((error) => {
```

Comment:
Should we return promise in order to make `Promise.all` working properly?

```suggestion
        return extension.activated.catch((error) => {
```

Reply:
Completely agree!

Reply:
Oh, actually this is fine already, there is no extra brackets from top `{}`.